components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -259,24 +262,11 @@ - (CGPoint)targetPoint {
   }
 }
 
-- (void)sheetBehaviorDidUpdate {
-  // If sheet has been dragged off the bottom, we can trigger a dismiss.
-  if (self.sheetState == MDCSheetStateClosed &&
-      CGRectGetMinY(self.sheet.frame) > CGRectGetMaxY(self.bounds)) {
-    [self.delegate sheetContainerViewDidHide:self];
```

Comment:
did we lose some functionality? we are telling a delegate that the we finished. How does a client listen to that event.

Reply:
This is a private delegate so public functionality isn't being lost.